clang/test/Driver/sycl-no-rdc-fat-archive-win.cpp

```diff
@@ -0,0 +1,31 @@
+/// test behaviors of passing a fat static lib with -fno-sycl-rdc on Windows
+// REQUIRES: system-windows
+
+// Build a fat static lib that will be used for all tests
+// RUN: echo "void foo(void) {}" > %t1.cpp
+// RUN: %clangxx -target x86_64-pc-windows-msvc -fsycl %t1.cpp -c -o %t1_bundle.o
+// RUN: llvm-ar cr %t_lib.a %t1_bundle.o
+// RUN: %clang -### -fsycl -fno-sycl-rdc -fsycl-device-code-split=off --sysroot=%S/Inputs/SYCL %t_lib.a 2>&1 -ccc-print-phases | FileCheck %s
+// RUN: %clang -### -fsycl -fno-sycl-rdc -fsycl-device-code-split=auto --sysroot=%S/Inputs/SYCL %t_lib.a 2>&1 -ccc-print-phases | FileCheck %s
+// RUN: %clang -### -fsycl -fno-sycl-rdc -fsycl-device-code-split=per_kernel --sysroot=%S/Inputs/SYCL %t_lib.a 2>&1 -ccc-print-phases | FileCheck %s
+// RUN: %clang -### -fsycl -fno-sycl-rdc -fsycl-device-code-split=per_source --sysroot=%S/Inputs/SYCL %t_lib.a 2>&1 -ccc-print-phases | FileCheck %s
```

Comment:
Any way we can throw in some `%clang_cl` type of test in the mix?

Reply:
Good catch, the option didn't work for clang_cl. I fixed this and added tests for it in my latest commit. I'll rename the PR to capture it has a source change